src/payment/models/bank-tx/bank-tx.service.ts

```diff
@@ -16,8 +20,38 @@ export class BankTxService {
     private readonly bankTxBatchRepo: BankTxBatchRepository,
     private readonly buyCryptoService: BuyCryptoService,
     private readonly mailService: MailService,
+    private readonly settingService: SettingService,
+    private readonly frickService: FrickService,
+    private readonly olkyService: OlkypayService,
   ) {}
 
+  // --- TRANSACTION HANDLING --- //
+  @Interval(60000)
+  async checkTransactions(): Promise<void> {
+    try {
+      const settingKey = 'lastBankDate';
+      const lastModificationTime = await this.settingService.get(settingKey, new Date(0).toISOString());
+
+      const olkyTransactions = await this.olkyService.getOlkyTransactions(lastModificationTime);
+      const frickTransactions = await this.frickService.getFrickTransactions(lastModificationTime);
+
+      if (!olkyTransactions && !frickTransactions) return;
+
+      for (const bankTx of [...olkyTransactions, ...frickTransactions]) {
+        try {
+          await this.create(bankTx);
+        } catch (e) {
+          if (!(e instanceof ConflictException)) console.error(`Failed to import transaction:`, e);
+        }
+      }
```

Comment:
If we use the same settingKey for multiple banks, we have to break, if any request fails (because we might loose some transactions). It then could be:
```ts
const transactions = await Promise.all([
  this.olkyService.getOlkyTransactions(lastModificationTime),
  this.frickService.getFrickTransactions(lastModificationTime),
]).then(([olky, frick]) => olky.concat(frick));

for (const bankTx of transactions) {
  try {
    await this.create(bankTx);
  } catch (e) {
    if (!(e instanceof ConflictException)) console.error(`Failed to import transaction:`, e);
  }
}
```